### DISCUSSION TITLE
Link favourites to a track

### DISCUSSION THREAD

**Discussion by Hamoen72** [osmandapp/OsmAnd#18403] [Ideas]
It would be nice if favourites (waypoints) can be save in or linked to a track.
If I plan a route I would like to add points of interests to that route (e.g. restaurants, museums, viewpoints etc.). They should appear if the route is set to be visible. 
Thanks in advance.

**Comment by DmitryAlexei** [comment]
@Hamoen72 you can add a waypoint to a track using Context menu:
tap on a POI or long press any place on the map > in the Context menu tap Actions > Add track waypoint > choose your track

For more information you can see documentation for waypoint ( https://osmand.net/docs/user/personal/tracks#add-waypoint )

**Reply by ribena75** [reply]
I can't see how to get this to work (iOS + OsmAnd Pro) - see my screen recording.

https://github.com/osmandapp/OsmAnd/assets/9573646/4dbe055c-6928-4b97-b971-c90e30edc2e5

**Comment by DmitryAlexei** [comment]
- please update OsmAnd to the latest version
- display one or few tracks on the map 
- create at least one favorite, or you can use POI instead
- tap a favorite or a POI and in the point context menu > actions choose "add waypoint" option
- select a track to which this waypoint will be added
also see video



https://github.com/osmandapp/OsmAnd/assets/116079541/3ca0b1b5-179f-4511-9930-7701b39568a3

**Reply by ribena75** [reply]
Thanks. I'm on the latest version for iOS but I do not have that option in the actions menu at all.

Uploading RPReplay_Final1715672347.mp4…
